default.nix

```diff
@@ -0,0 +1,10 @@
+# for passing build options, see nix/README.md
+{ config ? { } }:
```

Comment:
I don't think we need this file at all right now, since we're not building any targets.